README.md

```diff
@@ -127,127 +127,205 @@ future state of the libraries and to discover projects for contribution.
-through the process. They can decide when a final comment period is triggered.
->
-> On the other hand, if the submitter is a core team member, they will identify
-another core team member, with consent, as their 'shepherd'. The shepherd would
-be the first contact for brainstorming, process and reviews. The core team
-would defer to the shepherd to do the first few rounds of reviews, after which
-the rest of the team should be engaged.
-
-## RFC Life Cycle
+To kick off an RFC process, simply create a [new tracking issue] and follow the
+instructions in the template. The issue includes a checklist of the various
+stages an RFC goes through.
+
+[new tracking issue]: https://github.com/aws/aws-cdk-rfcs/issues/new?assignees=&labels=management%2Ftracking%2C+status%2Fproposed&template=tracking-issue.md&title=proposal+title
+
+This section describes each stage in detail, so you can refer to it for
+guidance.
+
+### 1. Create a tracking issue
+
+Each RFC has a GitHub issue which tracks it from start to finish. The issue is
+the hub for conversations, community signal (+1s) and the issue number is used
+as the unique identifier of this RFC.
+
+> Before creating a tracking issue, please search for similar or related ideas in
+the RFC table above or in the issue list of this repo. If there is a relevant
+RFC, collaborate on that existing RFC, based on its current stage.
+
+Our [tracking issue template] includes a checklist of all the steps an RFC goes
+through and it's the driver's responsibility to update the checklist and assign
+the correct label to on the RFC throughout the process.
+
+[tracking issue template]: https://github.com/aws/aws-cdk-rfcs/blob/master/.github/ISSUE_TEMPLATE/tracking-issue.md
+
+When the issue is created, it is required to fill in the following information:
+
+1. **Title**: the name of the feature or change - think changelog entry.
+2. **Description**: a _short_ description of feature, as if it was already implemented.
+3. **Proposed by**: fill in the GitHub alias of the person who proposed the idea
+   under "Proposed by".
+
+### 2. API Bar Raiser
+
+The public API of a feature represents the surface through which users interact
+with it, and we want to make sure these APIs are consistent, ergonomic and
+designed based on the intent and the mental model of our users. Additionally,
+once we announce that a feature is "stable" (1.0, GA, etc) any breaking change
+to its public API will require releasing a new major version, so we think of API
+decisions as "one way doors".
+
+For each RFC, the core team will assign an **API Bar Raiser** who reviews and
+approves any public API of the feature. API Bar Raisers have veto rights on
+API-related design decisions, such as naming, structure, options, CLI commands
+and others.
+
+After an RFC is proposed, the core team will assign it an API Bar Raiser using a
+tiering model generally based on the size of the user base that will likely get
```

Comment:
Where do you plan to document the tiering model and how they're determined?

Reply:
Our internal team flow documents